nltk/parse/dependencygraph.py

```diff
@@ -282,19 +282,24 @@ def _parse(self, input_, cell_extractor=None, zero_based=False, cell_separator=N
 
         def extract_3_cells(cells):
             word, tag, head = cells
-            return word, word, tag, tag, '', head, ''
+            return None, word, word, tag, tag, '', head, ''
```

Comment:
I've been thinking of this, problem, maybe it's better to make extractors to have an  argument `index`, with which the index from the loop below is passed, then it's up to the function to use it or not. something like:

```python
def extract_3_cells(cells, index):
    word, tag, head = cells
    return index, word, word, tag, tag, '', head, ''

...

def extract_10_cells(cells, index):
    index, word, lemma, ctag, tag, feats, head, rel, _, _ = cells
    return index, word, lemma, ctag, tag, feats, head, rel
```